### ISSUE TITLE
Partial importing of utils doesn't work

### ISSUE THREAD

**Issue by bartveneman** [csstree/csstree#181]
Inspired by [this MR](https://github.com/bramus/js-specificity/pull/4). To [reduce the bundle size](https://github.com/projectwallace/css-analyzer/pull/217) of my library, I want to import `property` from utils, but I get an error:

```
Error [ERR_MODULE_NOT_FOUND]: 
  Cannot find module '/Users/bart.veneman/www/css-analyzer/node_modules/css-tree/lib/utils/index.js' 
  imported from /Users/bart.veneman/www/css-analyzer/src/index.js
```

It seems that the lib/utils folder is missing an index file? Partially importing parse and walk do work.

## Steps to reproduce

```js
import utils from 'csstree/utils'
// also tried
import { getPropertyDescriptor } from 'csstree/utils'
```

## Workaround

```js
import { property } from '../node_modules/css-tree/lib/utils/names.js'
```

**Comment by bramus** [comment]
For the same reason [Skypack](https://skypack.dev/) can't build the package.

When trying to import CSSTree from https://cdn.skypack.dev/css-tree/css-tree, this error is logged:

```
[Package Error] \"css-tree@v2.0.4\" could not be built.
[1/5] Verifying package is valid…
[2/5] Installing dependencies from npm…
[3/5] Building package using esinstall…
Running esinstall...
ENOENT: no such file or directory, open 'css-tree/lib/utils/index.js'
ENOENT: no such file or directory, open '/tmp/cdn/_AMfY1QQhOrAHXZYTPCvI/node_modules/css-tree/lib/utils/index.js'
```

**Comment by lahmatiy** [comment]
Thank you for the report, I'm working on it

**Comment by bartveneman** [comment]
That was pretty impressive, thanks a bunch 😁

**Comment by lahmatiy** [comment]
Fixed in [2.1.0](https://github.com/csstree/csstree/releases/tag/v2.1.0)
